src/MainAppBar.js

```diff
@@ -56,7 +57,6 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 export default function MainAppBar(props) {
-  const {window} = props;
```

Comment:
The references to `window` may be required.  This was added by @ronaldewatts  during the responsive page updates.  I was adding the prop type validation for window and then I got curious how `window` was provided by a prop.  I tried to execute a scenario that I thought would make use of the prop but was unable to do that.  I might be wrong though.

Reply:
Came from the example, https://material-ui.com/components/drawers/#drawer.  I didn't question it :)